src/entities/DealTokenSwap.ts

```diff
@@ -960,7 +960,7 @@ export class DealTokenSwap implements IDeal {
 
       token.claimingClaimed = this.getClaimedAmount(dao, token.address);
       if (tokenClaimableAmounts.size > 0) {
-        token.claimingClaimable = tokenClaimableAmounts.get(token.address);
+        token.claimingClaimable = tokenClaimableAmounts.get(token.address) ?? BigNumber.from(0);
```

Comment:
left side could be `undefined`.
I did not dig deeper why, because it seems it should not be allowed to be `undefined` in the first place

Reply:
good catch.
_Typescript strict mode would help us catch these bugs early_